src/components/button/__styles__/index.ts

```diff
@@ -208,7 +208,7 @@ export const ElButton = styled.button`
   }
 `
 
-export const ElButtonGroup = styled.div`
+export const DeprecatedElButtonGroup = styled.div`
```

Comment:
**todo:** can you please add `/** @deprecated */` JS Doc to this component so that we get strike-through visual behaviour in our IDEs 🙏